documentation/Structure.md

```diff
@@ -0,0 +1,90 @@
+
+# Structure
+
+## Repositories
+
+    |--tutorial-compiler
+    |--tutorials
+
+## Tutorials
+
+    |-- tutorials
+        |-- [playbook-name]
+            /--index.asciidoc
+
+Write your own tutorials in "./tutorials/[playbook-name]/index.asciidoc".
+You will find all the details in the tutorials [Developement](https://github.com/devonfw-forge/tutorials/wiki/Development) page.
```

Comment:
maybe add playbook name should be similar like the branch in the tutorials-compiler for the CI build runs ?